code/game/objects/items/defibrillator.dm

```diff
@@ -66,14 +66,18 @@
 	playsound(get_turf(src), "sparks", 25, 1, 4)
 	update_icon()
 
+
 /mob/living/carbon/human/proc/get_ghost()
-	if(mind && !client) //Let's call up the correct ghost! Also, bodies with clients only, thank you.
-		for(var/mob/dead/observer/G in GLOB.player_list)
-			if(G.mind == mind)
-				var/mob/dead/observer/ghost = G
-				if(ghost && ghost.client && ghost.can_reenter_corpse)
-					return ghost
-	return 0
+	if(!mind || client) //Let's call up the correct ghost!
```

Comment:
See my comment on the bug report, this won't work.

Reply:
Yep, the issue is deeper. Was waiting on your input. This PR only refactors the currently non-working proc.

The problem is the lack of a reference to the ghost in the mob at the moment.
Suggestions?

Reply:
Like I said. remove the mind check, check if ghost.current == src